filter-entities.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// entities.json is from https://html.spec.whatwg.org/entities.json
+// which is built by https://github.com/whatwg/html-build/tree/main/entities
+// this processes it into a form more useful for us
+// no new entities will be added, so the output of this file is committed
+// rather than running it every time
+
+let fs = require('fs');
+let entities = require('./entities.json');
+
+let transformed = Object.fromEntries(Object.entries(entities).map(([k, v]) => {
```

Comment:
You could `flatMap` here to omit them instead of mapping them to `null`.

Reply:
I wanted to keep the keys present so I could use the same file to do https://github.com/tc39/ecmarkup/issues/476.